docs/protocols/authentication-specification.md

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 7
+title: Authentication
+---
+
+# Authentication Specification
+
+AdCP supports multiple authentication methods with consistent principal identification across protocols.
+
+## Authentication Methods
+
+### JWT Bearer Token
+```http
+Authorization: Bearer <jwt_token>
+```
+
+JWT tokens must include AdCP claims:
+```json
+{
+  "sub": "principal_123",
+  "adcp": {
+    "principal_type": "advertiser",
```

Comment:
I'm not sure principal_type matters - this is actually blurry in real life. I say we remove this concept entirely from the spec.